.github/workflows/testing.yml

```diff
@@ -30,39 +30,39 @@ jobs:
     name: Test run action (valid with custom Flake8)
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - run: pip install flake8==3.8.0
+    - uses: actions/checkout@v3
+    - run: pip install flake8==4.0.1
     - uses: ./
       with:
         path: example_valid
 
   run_action_valid_flake8_version:
-    name: Test run action (valid with Flake 3.8.0)
+    name: Test run action (valid with Flake 4.0.1)
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
     - uses: ./
       with:
         path: example_valid
-        flake8_version: 3.8.0
+        flake8_version: 4.0.1
 
-  run_action_valid_py39:
-    name: Test run action (valid with Python 3.9)
+  run_action_valid_py310:
+    name: Test run action (valid with Python 3.10)
```

Comment:
Here too, the version itself isn't important, as long as it isn't the version the runner is using.